lemur/plugins/lemur_gcp/plugin.py

```diff
@@ -0,0 +1,99 @@
+from flask import current_app
+from google.cloud.compute_v1.services import ssl_certificates
+from google.oauth2 import service_account
+import google.auth
+import hvac
+import os
+
+from lemur.plugins.bases import DestinationPlugin
+from lemur.plugins import lemur_gcp as gcp
+
+
+class GCPDestinationPlugin(DestinationPlugin):
+    title = "GCP"
+    slug = "gcp-destination"
+    version = gcp.VERSION
+    description = "Allow the uploading of certificates to GCP"
+    author = "Mitch Cail"
+    author_url = "https://github.com/Datadog/lemur"
+
+    options = [
+        {
+            "name": "Account Id",
+            "type": "str",
+            "required": True,
+            "helpMessage": "GCP Project ID",
+        },
+        {
+            "name": "Service Account Name",
+            "type": "str",
+            "required": True,
+            "helpMessage": "GCP Service Account Name",
+        },
+        {
+            "name": "Use Vault",
```

Comment:
Suggest calling this `authenticationMethod` with valid enum values of `vault` or `tokenFile` to be consistent with patterns established by other plugins. ([example](https://github.com/DataDog/lemur/blob/master/lemur/plugins/lemur_vault_dest/plugin.py#L55))